libraries/cmake/source/thrift/README.md

```diff
@@ -27,42 +29,48 @@ cmake \
   -B b \
   -DBUILD_SHARED_LIBS=OFF \
   -DBoost_USE_STATIC_LIBS=ON \
+  -DBoost_INCLUDE_DIR=../../../../../../cmake/source/boost/src/libs/config/include/ \
```

Comment:
Sorry I noticed now, I think this should be
```suggestion
  -DBoost_INCLUDE_DIR=/path/to/osquery/libraries/cmake/source/boost/src/libs/config/include/ \
```

Reply:
Ah, yes, if we prefer a placeholder path rather than a relative one that might work today and not tomorrow when things move. I'll change it.

Reply:
An alternative could've been having to set an env var to point the osquery source code too, but placeholder is fine too.